### DISCUSSION TITLE
Claims in the client_credentials flow

### DISCUSSION THREAD

**Discussion by projas-smartlog** [simpleidserver/SimpleIdServer#769] [Q&A] [answered]
Hi, I have a client in Postman of type client_credentials that uses the Scope 'api' of type API Resource. In this context, how do I ensure that the 'sub' claim (id or nameidentifier) is included in the token, because it's not there. Is this possible? 

![image](https://github.com/simpleidserver/SimpleIdServer/assets/173396359/96c30343-a0f7-473c-965e-4f3f63642979)

When the scope is set as API Resource, you cannot map claims like you can with Identity Resource type.

Thanks.

**Comment by simpleidserver** [comment, marked as answer]
Hello,

It is normal that the user sub is not returned in the claim because you are using the client_credentials grant type, and there is no user involved.
